src/mlagility/analysis/analysis.py

```diff
@@ -8,6 +8,7 @@
 import functools
 import dataclasses
 import pathlib
+import traceback
```

Comment:
Great contribution! Please set `MLAGILITY_TRACEBACK` to `True` on out Github Workflow tests

Reply:
Also, please add `MLAGILITY_TRACEBACK` and `MLAGILITY_DEBUG` to the user guide.

Reply:
Done and done!